src/BSeries.jl

```diff
@@ -612,6 +615,74 @@ function bseries(ros::RosenbrockMethod, order)
     return series
 end
 
+"""
+    ContinuousStageRungeKuttaMethod
+
+A struct that describes a CSRK method. This kind of 'struct' should be constructed 
+via [`CSRK`] 
+        'csrk = CSRK(M)' 
+in order to later call the 'bseries' function. 
+"""
+struct ContinuousStageRungeKuttaMethod{MatT <: AbstractMatrix}
+    matrix::MatT
+end
+
+function CSRK(matrix::AbstractMatrix)
+    T = promote_type(eltype(matrix))
+    _M = T.(matrix)
+    return ContinuousStageRungeKuttaMethod(_M)
+end
+
+"""
+        bseries(csrk::ContinuousStageRungeKuttaMethod, order)
+
```

Comment:
```suggestion
"""
    bseries(csrk::ContinuousStageRungeKuttaMethod, order)

```
Indentation with 4 space to mark it as code. With 8 spaces, it will be indented code as in

```
  bseries(ros::RosenbrockMethod, order)

  Compute the B-series of the Rosenbrock method ros up to a prescribed integer order.

  │ Normalization by elementary differentials
  │
  │  The coefficients of the B-series returned by this method need to be multiplied by a power of the
  │  time step divided by the symmetry of the rooted tree and multiplied by the corresponding
  │  elementary differential of the input vector field f. See also evaluate.

  ────────────────────────────────────────────────────────────────────────────────────────────────────────────

      bseries(csrk::ContinuousStageRungeKuttaMethod, order)

  Return a truncated B-series up to the specified order with coefficients determined by the square matrix
  located in csrk. The coefficients for every RootedTree are obtained via the 'elementarydifferentialscsrk'
  function, which is calculated according to Miyatake & Butcher (2015). [See @ref csrk]
```